### PR TITLE
Desktop version branch

### PR DESCRIPTION
**Pull request by GhostEsso**
Pull Request Description:

Added Linters
Desktop version added
Readme file is corrected
**

- I gave the button some padding-bottom so as to this section match the provided design
- I gave the section the same padding at the top on both the left side and right side to make it look the same as the Figma provided
- I increased the margin i gave this project-card class so as to make my Desktop version looks like the one in the design

Thank you for reviewing my Pull Request

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -284,16 +282,19 @@ <h2 class="header-text">About <br>Myself</h2>
...
-
-
-
-
-
-
-
-
-
-
```
Comment (newhoteng) [inline index.html]: I think you should delete the empty lines.

**Review by BranBayou — COMMENTED** [review]
# Change Required :recycle: 
Please fix the conflict your style.css file has. Your pull request needs to be free from conflicts.

**Review by aamirkhan2478 — COMMENTED** [review]
Hi @GhostEsso , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that hasn't a PR description please add your PR description and then resubmit for review your PR description should be descriptive and meaningful. Check below :point_down: example of descriptive PR description
184990532-da778ab4-c789-4136-90a9-629eedd969da



Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: style.css [hunk 2]
```diff
@@ -743,3 +743,270 @@ form {
...
+  .nav-container {
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding-left: 10px;
+    padding-right: 10px;
+    margin-top: 26px;
+    height: 10vh;
+  }
```
Comment (divinecharlotte) [inline style.css]: - [ ] kindly consider removing the margin-top you gave this nav-container so as to implement the Figma design
- 
rimw

File: style.css [hunk 3]
```diff
@@ -743,3 +743,270 @@ form {
...
+  .list-nav {
+    margin-top: 20px;
+    margin-left: -9%;
+  }
+
+  ul.list-items {
+    list-style: none;
+    margin: -25px;
+    padding-top: 10px;
+  }
```
Comment (divinecharlotte) [inline style.css]: - [ ]  kindly consider moving this ul up a bit to make the fit inside the background image

File: index.html [hunk 4]
```diff
@@ -164,11 +162,11 @@ <h2 class="project-title">Tonic</h2>
             <li>javascript</li>
           </ul>
         </div>
-        <button class="see-project-btn" type="button">See Project</button>
+        <button class="see-project-btn-1" type="button">See Project</button>
       </div>
     </div>
     <div class="project-card">
```
Comment (divinecharlotte) [inline index.html:168]: - [x] kindly consider increasing the margin you are giving this  project-card class so as to make your   Desktop version looks like the one in the design 
- 
kabi

File: index.html [hunk 5]
```diff
@@ -285,19 +283,22 @@ <h2 class="header-text">About <br>Myself</h2>
       </div>
     </div>
```
Comment (divinecharlotte) [inline index.html:284]: - [x] kindly consider giving this section the same padding at the top on both the left side and right side to make it look the same as the Figma provided
- your design 👇 
gata

- Figma design 👇 
gatfi

File: index.html [hunk 6]
```diff
@@ -285,19 +283,22 @@ <h2 class="header-text">About <br>Myself</h2>
...
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+      </div>
+      <form action="https://formspree.io/f/myyawnzb" method="post">
+        <input class="input-1" type="text" placeholder="Yeremias" maxlenght="30" required>
+        <input class="input-2" type="email" name="email" placeholder="natajayanj14@gmail.com" maxlength="30" required>
+        <textarea class="input-3" name="textarea" id="" cols="30" rows="10" maxlength="500" placeholder="Write your message here"></textarea>
+        <button id="btn">Get in touch</button>
+        <img class="footer-shape" src="./images/Shape.svg" alt="">
+      </form>
+    </section>
```
Comment (divinecharlotte) [inline index.html:299]: - [x] kindly consider giving this button some padding-bottom so as to this section match the provided design 
- 
pad

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
# Hello @GhostEsso      👋 , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Highlights 🍾 
- GitHub Flow observed ✔️
- Good commit messages ✔️
- Descriptive Readme 👍 
## Required Changes ♻️
consider the comments under the codes

## Optional suggestions 💭 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MeqdamAlqudah — APPROVED** [review]
Hi @GhostEsso , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
image

## Highlights 
:+1: no linter errors 
:+1: all comments by the last reviewer were fixed.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**